docs/reference/remap.cue

```diff
@@ -176,6 +176,20 @@ remap: #Remap & {
 				}
 			}
 		},
+		{
+			title: "Emitting multiple logs from JSON"
```

Comment:
I plan to add more interesting examples once the `unnest` function part of #7038 is implemented. I might write up a quick guide too, at that point.

Reply:
I attempted to add this example, but it didn't quite work out as:

* The VRL tests in `lib/vrl/tests` that test these examples, when testing output, simply add `; .` to the end of the VRL programs and evaluate the return value of executing the program
* The conversion of non-object values to `{ "message": <value> }` actually happens _after_ the VRL program runs, in the `remap` transform

So the example would need to return the primitives, rather than the structured log events, which would be confusing since they are converted to log events.

I think, ideally, the examples in `docs/reference/remap.cue` would be tested at a different level than `lib/vrl/tests` where it would actually run the `remap` transform; including its final conversion. That's a bit more than I'd like to bite off for this PR though so I might defer for now if that's ok. I'll add a test for this to `tests/behavior/transforms/remap.toml` though.

Reply:
Yeah, I agree that might be the best solution in the end. However, I think we can still add a test here for documentation purposes, but ignore it in the actual test run itself, at least for now.

Reply:
👍 I added in 563a627ba